content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -111,17 +115,106 @@ $ kubectl apply -f @samples/sleep/notsleep.yaml@
 
 Note: `sleep` and `notsleep` are two simple applications that can serve as curl clients.
 
-Connect `productpage` to the Istio ingress gateway so you can access the bookinfo
-app from outside of the cluster:
+{{< tabset category-name="config-api" >}}
+
+{{< tab name="Istio classic" category-value="istio-classic" >}}
+
+Create an Istio [Gateway](/docs/reference/config/networking/gateway/) and
```

Comment:
do they need to install the deployment as well? I think we turned it off by default in ambient (or did we not yet?)

Reply:
Do you mean that the intent is to turn off managed gateways in ambient mode? Since the test is passing, I assume that isn't the case currently, but is that going to change?

Reply:
No I mean we don't auto-deploy an istio-ingressgateway deploymnent in `istioctl install --set profile=ambient`. But maybe that was a "we should do this" not "we did".

Reply:
> do they need to install the deployment as well? I think we turned it off by default in ambient (or did we not yet?)

@howardjohn a deployment does not need to be cerated by the user since the Gateway auto creates the deployment and service. Note that the Gateway in this doc is not running on ambient (istio-system ns).

> No I mean we don't auto-deploy an istio-ingressgateway deploymnent in istioctl install --set profile=ambient

This would be a separate change to the doc since this PR is for adding Gateway API steps. Istio is still deploying ingress gateways for the Ambient profile:
```
$ echo $TAG
1.18-alpha.42c42563c082e6e4b457d5b94448be9cd6c43f26

$ istioctl install --set profile=ambient --set hub=$HUB --set tag=$TAG
This will install the Istio 1.18.0 ambient profile with ["Istio core" "Istiod" "CNI" "Ztunnel" "Ingress gateways"] components into the cluster. Proceed? (y/N) y
✔ Istio core installed
✔ Istiod installed
✔ Ingress gateways installed
✔ CNI installed
✔ Ztunnel installed
✔ Installation complete                                                                                                                                         Making this installation the default for injection and validation.
```

I created https://github.com/istio/istio/issues/44030 to address this issue but I don't see a PR against it. If you'd like me to work on it, please assign me.

Reply:
>a deployment does not need to be cerated by the user since the Gateway auto creates the deployment and service. Note that the Gateway in this doc is not running on ambient (istio-system ns).

I was specifically commenting on the "istio classic" tab

Reply:
Sorry for creating the confusion ... I didn't notice this comment was on the classic instruction :(

@howardjohn I thought that the ambient profile is temporary and will be removed in the future when ambient mode will be enabled by default. If that's true, users would use the minimal profile if they don't want the ingress gateway installed, same as non-ambient mode. Maybe, at some time in the future, the minimal profile will become default, but that seems like a breaking change?

Reply:
> If that's true, users would use the minimal profile if they don't want the ingress gateway installed, same as non-ambient mode.

@frankbu ^ users would need to manually install CNI and ztunnel components. When https://github.com/istio/istio/issues/44030 is fixed, the ambient profile will install istiod, cni, and ztunnel. Users can then create an Istio Gateway/VirtualService or a k8s Gateway/HTTPRoute for ingress.

Reply:
> @frankbu ^ users would need to manually install CNI and ztunnel components.

By `If that's true, ...` I meant if we make everything in the current ambient profile enabled by default in the future.

> When https://github.com/istio/istio/issues/44030 is fixed, the ambient profile will install istiod, cni, and ztunnel. Users can then create an Istio Gateway/VirtualService or a k8s Gateway/HTTPRoute for ingress.

Not quite. If they want to use Gateway/VirtualService, they will also need to manually deploy the istio-ingressgateway.